app/assets/javascripts/discourse/tests/acceptance/sidebar-plugin-api-test.js

```diff
@@ -21,7 +19,6 @@ acceptance("Sidebar - Plugin API", function (needs) {
   });
 
   needs.hooks.afterEach(() => {
-    resetSidebarSection();
```

Comment:
Doesn't look like this has been introduced elsewhere... is it not needed at all?

Reply:
It already being done after each test:

https://github.com/discourse/discourse/blob/e21d2ac217e146005d8454b08f2de1e43d6dc3de/app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js#L203-L203